src/newbank/server/banker.java

```diff
@@ -0,0 +1,21 @@
+package newbank.server;
+
+import java.util.ArrayList;
+
+public class banker extends User {
```

Comment:
Preferred to have Title case for Classes

Reply:
Agree.